src/lesson/lesson03.js

```diff
@@ -1346,11 +1427,11 @@ function Lesson03() {
       ),
       "",  // initialCode
       Constants.Lesson03.SUCCESS_MESSAGE,
-      null
+      22
```

Comment:
Again, this has a trivial solution without conditionals: FFLGRFFFFLPRFLGRFLP within this limit of 22 characters. A smaller limit can prevent this.